provisioning/utils.go

```diff
@@ -59,25 +59,25 @@ func getPodHostIP(podClient coreclientv1.PodsGetter, targetNamespace string) (st
 	return hostIP, err
 }
 
-func getServerInternalIP(osclient osclientset.Interface) (string, error) {
+func getServerInternalIP(osclient osclientset.Interface) ([]string, error) {
 	infra, err := osclient.ConfigV1().Infrastructures().Get(context.Background(), "cluster", metav1.GetOptions{})
 	if err != nil {
 		err = fmt.Errorf("Cannot get the 'cluster' object from infrastructure API: %w", err)
-		return "", err
+		return []string{""}, err
```

Comment:
We used "" as an empty value for the `string` type. The slice type has an empty value as well - `nil`, so let's return it. It's especially true for the error case.

Reply:
In case of an error should we fall back to Pod IP ( as is the case for NonePlatform ) or simply return ?